parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -214,6 +214,16 @@ function Register-NodeResetScriptTask {
     Register-ScheduledTask -TaskName "k8s-restart-job" -InputObject $definition
 }
 
+function Register-NodeLabelSyncScriptTask {
+    Write-Log "Creating a periodical task to run windowsnodelabelsync.ps1"
+
+    $action = New-ScheduledTaskAction -Execute "powershell.exe" -Argument "-File `"c:\k\windowsnodelabelsync.ps1`""
+    $principal = New-ScheduledTaskPrincipal -UserId SYSTEM -LogonType ServiceAccount -RunLevel Highest
+    $trigger = New-JobTrigger -AtStartup -RandomDelay 00:01:00  -RepetitionInterval 00:01:00

```

Comment:
Add `RepeatIndefinitely`

Reply:
Sure. The current behavior is run indefinitely with fixed interval per my test.

Reply:
https://docs.microsoft.com/en-us/powershell/module/psscheduledjob/new-jobtrigger?view=powershell-5.1
```
-RepeatIndefinitely
This parameter, available starting in Windows PowerShell 4.0, eliminates the necessity of specifying a TimeSpan.MaxValue value for the RepetitionDuration parameter to run a scheduled job repeatedly, for an indefinite period.
```

Reply:
As I said, I will add this parameter. Just share my test result.

Reply:
Changed to `New-ScheduledTaskTrigger` instead.
I did not successfully create a trigger through `New-JobTrigger`, in my test, I used `New-ScheduledTaskTrigger` to create a trigger. sorry for taking your time to review the wrong one.